### ISSUE TITLE
Test `test_IXS_G_to_L` ocassionally fails

### ISSUE THREAD

**Issue by costrouc** [atztogo/phonopy#79]
I cannot reproduce every time (only ocassionally does the test not pass) but I am getting `test_IXS_G_to_L` to fail. Full error log is [here](https://logs.nix.ci/?key=nixos/nixpkgs.46161&attempt_id=58a6197d-2561-4402-b2f3-2abba869768c).
Here is snippet of the error message.

```
running install tests
..........................................F........
======================================================================
FAIL: test_IXS_G_to_L (spectrum.test_dynamic_structure_factor.TestDynamicStructureFactor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/phonopy-1.13.2.107/test/spectrum/test_dynamic_structure_factor.py", line 102, in test_IXS_G_to_L
    S[:, i].sum(axis=1), data_cmp[:, i].sum(axis=1), atol=1e-1)
  File "/nix/store/3aqb8gxmjy4bd98zh01as7nrcyszf3c2-python3.6-numpy-1.15.1/lib/python3.6/site-packages/numpy/testing/_private/utils.py", line 1452, in assert_allclose
    verbose=verbose, header=header, equal_nan=equal_nan)
  File "/nix/store/3aqb8gxmjy4bd98zh01as7nrcyszf3c2-python3.6-numpy-1.15.1/lib/python3.6/site-packages/numpy/testing/_private/utils.py", line 789, in assert_array_compare
    raise AssertionError(msg)
AssertionError:
Not equal to tolerance rtol=1e-07, atol=0.1
(mismatch 10.0%)
 x: array([2.990558e+01, 8.017501e+00, 3.839426e+00, 2.339812e+00,
       1.633024e+00, 1.245578e+00, 1.107560e-03, 1.267848e-04,
       4.048384e-04, 2.625977e-01])
 y: array([2.990558e+01, 8.017501e+00, 3.839426e+00, 2.339812e+00,
       1.633024e+00, 1.245578e+00, 6.772900e-05, 3.625850e-04,
       2.829889e-02, 5.767737e-03])
----------------------------------------------------------------------
Ran 51 tests in 22.281s
``` 

I am trying to include `phonopy` in the nix package manager18.09 release. Here is the pull request that I have open https://github.com/NixOS/nixpkgs/pull/46161

**Comment by atztogo** [comment]
Thanks. I will have a look and fix it.

**Comment by atztogo** [comment]
I think it's fixed at 01ce38c46de54efe4966399efff02df05a92ab3, which is currently on develop branch.

**Comment by costrouc** [comment]
Thanks! For now I have just removed the specific test from the release. The test will be included at the next phonopy release.
